File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L4/06/looping.ts
import { Scanner } from 'readline-sync';

function print(mil: number): void {
  while (mil <= 10) {
    console.log(`${mil}\t\t${mil * 1.609}`);
    mil++;
  } // looping dari 1 sampai 10
}

function main(): void {
  console.log('Miles\t\tKilometers');
  console.log('-------------------------------');
  let mil: number;
  print((mil = 1));
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L4/06/looping.kt
package IR

import java.util.Scanner

fun main() {
    println("Miles\t\tKilometers")
    println("-------------------------------")
    var mil = 1
    print(mil)
}

fun print(mil: Int) {
    var milVar = mil
    while (milVar <= 10) {
        println("$milVar\t\t${milVar * 1.609}")
        milVar++
    } // looping dari 1 sampai 10
}

